packages/ckeditor5-heading/src/title.js

```diff
@@ -122,13 +122,14 @@ export default class Title extends Plugin {
 	 * data output, like comments or track changes features. If such markers start in the title and end in the
 	 * body, the result of this method might be incorrect.
 	 *
+	 * @param {Object} [options] Additional configuration passed to the conversion process.
```

Comment:
Please link to `DataController#getData()` (see {@link...}) where, AFAIR, this parameter is also described. Same below.